lib/OpenQA/Utils.pm

```diff
@@ -1313,5 +1313,30 @@ sub any_array_item_contained_by_hash {
     return 0;
 }
 
+#replace %NAME% with $settings{NAME}
```

Comment:
A space after the `#` would look better, right? (Same counts for the other comments.)